type-o-matic/Program.cs

```diff
@@ -160,6 +264,148 @@ static CSMethod PInvokeForType (TypeDefinition type, Dictionary<string, string>
 					mangledName.Substring (1), new CSParameterList ());
 		}
 
+		static CSMethod PInvokeForName(string name, Dictionary<string, string> funcs)
```

Comment:
same minor whitespace in function declaration.

Reply:
Fixed